spec.go

```diff
@@ -290,6 +291,15 @@ var mountPropagationMapping = map[string]int{
 	"":         syscall.MS_PRIVATE | syscall.MS_REC,
 }
 
+// validateSpec validates the fields in the spec
+// TODO: Add validation for other fields where applicable
+func validateSpec(spec *specs.LinuxSpec, rspec *specs.LinuxRuntimeSpec) error {
+	if spec.Process.Cwd == "" {
```

Comment:
if https://github.com/opencontainers/specs/pull/312 is adopted before this PR is merged, can you add a check to ensure its an abs path?  `filepath.IsAbs()`